egs/wsj/s5/steps/nnet3/chain/build_tree.sh

```diff
@@ -66,9 +66,6 @@ cp $alidir/splice_opts $dir 2>/dev/null # frame-splicing options.
 cp $alidir/cmvn_opts $dir 2>/dev/null # cmn/cmvn option.
 cp $alidir/delta_opts $dir 2>/dev/null # delta option.
 
-utils/lang/check_phones_compatible.sh $lang/phones.txt $alidir/phones.txt || exit 1;
```

Comment:
is there a reason you deleted these lines?  Perhaps something went wrong in merge?